views/sections/library.html

```
@@ -62,3 +61,88 @@ <h1 class="text-3xl font-bold text-slate-800 dark:text-white">Music Library</h1>
+            <i id="player-icon" class="fas fa-play"></i>
+          </button>
+          <button type="button" title="Next" _="on click call playerStep(1)"
+                  class="w-9 h-9 flex items-center justify-center rounded-full text-gray-600 dark:text-gray-300 hover:bg-gray-100 dark:hover:bg-neutral-800 transition-colors">
+            <i class="fas fa-forward-step"></i>
+          </button>
+        </div>
+
+        <!-- Col 3: volume -->
+        <div class="flex items-center justify-end gap-2">
+          <i class="fas fa-volume-high text-gray-400 text-xs"></i>
+          <input type="range" min="0" max="100" value="100"
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)